clustermesh-apiserver/option/config.go

```diff
@@ -54,13 +54,16 @@ func (def LegacyKVStoreMeshConfig) Flags(flags *pflag.FlagSet) {
 }
 
 type KVStoreMeshConfig struct {
-	RemoteClusterReadyTimeout time.Duration
+	PerClusterReadyTimeout time.Duration
+	GlobalReadyTimeout     time.Duration
 }
 
 var DefaultKVStoreMeshConfig = KVStoreMeshConfig{
-	RemoteClusterReadyTimeout: 1 * time.Minute,
+	PerClusterReadyTimeout: 1 * time.Minute,
+	GlobalReadyTimeout:     2 * time.Minute,
```

Comment:
I'd personally raise this default to an higher value to account for large scale environments as well. Maybe something in the 5-10m range?